gef.py

```diff
@@ -7354,10 +7353,10 @@ def context_title(self, m: Optional[str]) -> None:
 
     def context_regs(self) -> None:
         self.context_title("registers")
-        ignored_registers = set(self["ignore_registers"].split())
+        ignored_registers = self["ignore_registers"].split()
 
         if self["show_registers_raw"] is False:
-            regs = set(gef.arch.all_registers)
+            regs = [x for x in gef.arch.all_registers if x not in ignored_registers]
```

Comment:
use `r` or `reg` instead of `x`

Reply:
We improve code when we touch it.

Reply:
Not disagreeing, have changed from `x` to `reg`